src/templates/static/about/job_details.jsx

```diff
@@ -106,473 +109,642 @@ const JobDetails = () => (
 
                             <UlText
-                                text={it.L('Duties and Responsibilities:')}
+                                text={it.L('Preferences:')}
                                 className='bullet'
                                 items={[
-                                    { text: it.L('Work directly with members of our marketing and IT teams to produce graphics for the company website, blog, social media outlets, and marketing materials') },
-                                    { text: it.L('Create content to support our international marketing campaigns in a variety of different formats and languages') },
+                                    { text: it.L('An advanced university degree in physics, financial engineering, or mathematics is preferred') },
+                                    { text: it.L('Experience in Forex spot trading or exotic options pricing, volatility forecasts, high-frequency trading, and the analysis of market inefficiencies') },
+                                    { text: it.L('Firm grasp of advanced risk management concepts, including hedging, Greeks (first and second generation), Value at Risk models, etc.') },
+                                    { text: it.L('Experience in using financial information sources such as Bloomberg and Reuters') },
+                                    { text: it.L('Strong analytical skills and the ability to communicate your findings in a clear, concise, and effective manner') },
+                                    { text: it.L('Relevant experience in the use of quant programming libraries and frameworks (QuantLib, Pricing Partners, FINCAD, and Numerix), and quant pricing platforms (SuperDerivatives and FENICS) would be a plus.') },
                                 ]}
                             />
+                        </div>
+                    </div>
+
+                    <div className='marketing'>
+                        <div className='invisible' id='affiliate_country_manager'>
+                            <h1>{it.L('Affiliate/Country Manager')}</h1>
+
+                            <p>{it.L('The Country Manager is expected to acquire, service, manage, and expand our network of active affiliates in Russian-speaking countries. You will also contribute your energy, skills, and knowledge of the local business culture to increase our rapid growth rate worldwide.')}</p>
+                            <p>{it.L('You are also expected to help us promote our cutting-edge trading platform that has been one of the most recognised in the binary options trading industry for over 18 years.')}</p>
 
                             <UlText
-                                text={it.L('Requirements:')}
+                                text={it.L('Responsibilities:')}
                                 className='bullet'
    
```

Comment:
please use a placeholder for `<a href=\'https://www.binary.com\' target=\'_blank\'>www.binary.com</a>` so we don't send this to translation